sdk/python3/README.md

```diff
@@ -90,17 +150,17 @@ container_measurement = Measurement.get_container_measurement()
 
 ### Event log
 
-Using this sdk, user can fetch the event logs to assist the attestation/verification process. It also enables two different categories of event logs - for the platform or for a single workload/container.
+Using this SDK, user can fetch the event logs to assist the attestation/verification process. It also enables two different categories of event logs - for the platform or for a single workload/container.
 From platform perspective, it can support different Trusted Execution Environment and TPM. This sdk can also do fetching on certain number of event logs.
 
 ##### EventlogType for platform
```

Comment:
can you also fix this one for me? "#####"->"####"

Reply:
Done.